ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageSingleVulnerabilities.tsx

```diff
@@ -73,18 +74,7 @@ function ImageSingleVulnerabilities({ imageId }: ImageSingleVulnerabilitiesProps
     const vulnerabilityData = data ?? previousData;
 
     if (error) {
-        mainContent = (
-            <Bullseye>
-                <EmptyState variant={EmptyStateVariant.large}>
-                    <EmptyStateIcon
-                        className="pf-u-danger-color-100"
-                        icon={ExclamationCircleIcon}
-                    />
-                    <Title headingLevel="h2">{getAxiosErrorMessage(error)}</Title>
-                    <EmptyStateBody>Adjust your filters and try again</EmptyStateBody>
-                </EmptyState>
-            </Bullseye>
-        );
+        mainContent = <ErrorSection error={error}>Adjust your filters and try again</ErrorSection>;
```

Comment:
This was the third time I copy/pasted this block, so I figured it was time to extract it.